### PR TITLE
Feature 2556: Integrate cwe extended bandit sarif formatter plugin into pds-solutions

### LINKED ISSUE #2556: Integrate cwe extended bandit sarif formatter plugin into pds-solutions multi container
### Situation
Currently the Bandit Sarif report does not contain information about the Common Weakness Enumeration (CWE) taxonomy. As a developer I would like to have a docker environment for testing a future CWE Sarif integration into Bandit.

### Solution
Extend the docker multi container ( https://github.com/mercedes-benz/sechub/tree/develop/sechub-pds-solutions/multi ) with the CWE extended Bandit Sarif Plugin ( https://github.com/alexdd/bandit-sarif-formatter )

### PR DESCRIPTION
**Pull request by alexdd**
closes #2556 

Currently the CWE Sarif Plugin is located at my personal repository and will be fetched in the docker build https://github.com/alexdd/bandit-sarif-formatter

### CODE REVIEW

File: sechub-pds-solutions/multi/docker/Multi-Debian.dockerfile [hunk 1]
```diff
@@ -39,6 +39,11 @@ COPY packages.txt $TOOL_FOLDER/packages.txt
 # Interesting idea, but not as useful inside a container, which in essence is already a virtual environment.
 # Use `--break-system-packages` to let the Python package manager `pip` mix packages from Debian and Python
 RUN pip install --break-system-packages -r $TOOL_FOLDER/packages.txt
+# The pip -e option seems to be broken in recent versions of setuptools
+# when it comes to installing modules from the local filesystem,
+# see: https://github.com/pypa/setuptools/issues/3301
+# this is why we download the plugin using a GIT http call and not use GIT submodules
+RUN pip install --break-system-packages -e "git+https://github.com/alexdd/bandit-sarif-formatter#egg=bandit-sarif-formatter"
```
Comment (Jeeppler) [inline sechub-pds-solutions/multi/docker/Multi-Debian.dockerfile:46]: What does `-e` stand for? Is there a long form for `-e`? 
I would prefer to fix the git URL to a specific commit/tag, is that possible?

**Review by Jeeppler — CHANGES_REQUESTED** [review]
See comment.

File: sechub-pds-solutions/multi/docker/Multi-Debian.dockerfile (hunk 1 above)
Comment (alexdd) [reply sechub-pds-solutions/multi/docker/Multi-Debian.dockerfile]: Both is possible I updated the pull request. I won't put any more commits to the `alexdd/bandit-sarif-formatte`r repo, so `@main` should be fine (just for example).

**Review by Jeeppler — APPROVED** [review]
Approved 👍. Ready to be merged.

**Pull request merged**